apps/apps.go

```diff
@@ -20,9 +20,11 @@ import (
 	"tidbyt.dev/community/apps/dateprogress"
 	"tidbyt.dev/community/apps/datetimeclock"
 	"tidbyt.dev/community/apps/daynightmap"
+	"tidbyt.dev/community/apps/destiny2stats"
```

Comment:
The build failure is related to the changes here - you can simply add another commit that removes references to other apps to resolve.